src/tracers.fs

```diff
@@ -0,0 +1,45 @@
+namespace Elmish.Tracers
+
+open Elmish
+open System
+open Fable.Core
+
+[<RequireQualifiedAccess>]
+module Tracers =
+
+    let getMsgNameAndFields (t: Type) (x: 'Msg) : string * obj =
```

Comment:
this can be `internal` if `console` function bellow is not inlined (and I don't think it should be).